polymetis/polymetis/python/polymetis/robot_interface.py

```diff
@@ -744,3 +784,32 @@ def move_ee_xyz(
             "The method 'move_ee_xyz' is deprecated, use 'move_to_ee_pose' with 'delta=True' instead."
         )
         return self.move_to_ee_pose(position=displacement, delta=True, **kwargs)
+
+    # MIRROR METHODS
+
+    def sync_with_mirror(self):
+        assert self.use_mirror_sim, "Mirror sim must be instantiated!"
+        assert not self.mirror_sim_robot, "Clean mirror after forward in order to sync!"
+        self.mirror_sim_client.sync(self)
+
+    def unsync_with_mirror(self):
+        assert self.use_mirror_sim, "Mirror sim must be instantiated!"
+        self.mirror_sim_client.unsync()
```

Comment:
The user doesn't really need access to these methods. We should be able to unsync before running any policy and resync afterwards within `RobotInterface`.